pipeline-runner/src/data-ingest/0-download_gem2s.r

```diff
@@ -60,4 +60,6 @@ task <- function(input,pipeline_config) {
     message(exportJSON)
     write(exportJSON, "/input/meta.json")
     message('Written config json')
+
+    send_update_to_api(pipeline_config, experiment_id = input$experimentId, status_msg = "OK")
```

Comment:
Would a status message that included the task name make debugging from the API easier?